nfc_magic/scenes/nfc_magic_scene_gen4_select_shd_mode.c

```diff
@@ -0,0 +1,106 @@
+#include "../nfc_magic_app_i.h"
+#include "protocols/gen4/gen4_poller_i.h"
```

Comment:
Here and below please avoid including private API. If you need access to internal Gen4Poller fileds, add getters() in public API